interview-questions.md

```diff
@@ -44,15 +44,15 @@ Then, research the question to expand on your answer. Even if you feel you have
 
   Your answer: Object destructuring is like breaking variables down or being more specific.
```

Comment:
I like this breakdown. If you support it with and example of how you used it in class..you will on the start of a great interview.